spec/factories/digital_objects/items.rb

```diff
@@ -1,11 +1,26 @@
-# FactoryBot.define do
-#   factory :digital_object_item, class: DigitalObject::Item do
-#     initialize_with do
-#       instance = new
-#       instance.uid = 'abc-123'
-#       # TODO: Add other setup steps for this item factory
-#
-#       instance
-#     end
-#   end
-# end
+FactoryBot.define do
+  factory :item, class: DigitalObject::Item do
+    initialize_with do
+      instance = new
+      instance.instance_variable_set('@uid', Random.rand.to_s)
```

Comment:
I'd recommend changing `Random.rand.to_s` to `SecureRandom.uuid` to match our usual uid format.

Reply:
Sure. I'll have to fix it in the other fixture. In theory, we wouldn't need to set the uid regardless right?

Reply:
Oh yeah, you're right.  In that case, should we remove the whole `instance_variable_set ` line?

Reply:
Yea, i think so. I removed it from this fixture and the other digital object fixture.